src/tests/commonTests.ts

```diff
@@ -244,6 +298,15 @@ export async function slots(
   }
 }
 
+/**
+ * Helper for asserting that a component is labelable
```

Comment:
Nitpick: For readability, can you add a newline to visually separate the description from the params? I'll leave this up to you. 

On a related note, WDYT about using https://www.npmjs.com/package/eslint-plugin-jsdoc to help keep JSDoc in check?

Reply:
Sure, I'll add the newline and remove the jsdoc from the non-exported functions. And yeah the jsdoc eslint plugin sounds like a great idea! I added it to my todo list.